src/seedu/addressbook/data/person/Email.java

```diff
@@ -36,26 +33,4 @@ public Email(String email, boolean isPrivate) throws IllegalValueException {
     public static boolean isValidEmail(String test) {
```

Comment:
If you had designed the accurately as mentioned above, this method could have been pushed tot eh super class as well.